scripts/run_all_benchmarks.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+if [ "$1"="-h" -o "$1"="--h" -o "$1"="--help" ] ; then
+    echo "Usage:  $0 <project-home-dir>"
+fi
+
+export RUST_LOG=info
+PROJECT=${1:-$HOME/frequency}
+CHAIN=$PROJECT/res/genesis/testnet/frequency-spec-rococo-testnet.json
+RUNTIME=$PROJECT/target/release/frequency
+BENCHMARK="$RUNTIME benchmark pallet "
+OUTPUT_DIR=$PROJECT/runtime/frequency/src/weights
```

Comment:
Note this is not really correct for our repo; @enddynayn has a draft PR suggesting a single directory for weights, so treat this as a placeholder. I've been using the root directory for output and copying the weights into the current places.